offset_request.go

```diff
@@ -1,19 +1,7 @@
 package sarama
 
-// OffsetTime is used in Offset Requests to ask for all messages before a certain time. Any positive int64
-// value will be interpreted as milliseconds, or use the special constants defined here.
-type OffsetTime int64
-
-const (
-	// LatestOffsets askes for the latest offsets.
-	LatestOffsets OffsetTime = -1
```

Comment:
I'm ok with removing these - the only burden is on the users of the low-level API, which there are very few of (if any). However, we need to update the docs on the ones that are left in order to make them not consumer-specific anymore